faq.md

```diff
@@ -0,0 +1,99 @@
+ Sanitizer API - FAQs
+======================
+
+## How do I use the Sanitizer API
+
+Construct a new Sanitizer, using either the defaults or suppliying a custom
+configuration, and then pass in a string, a fragment, or an entire document
+for sanitization.
+
+```js
+const s = new Sanitizer();
+
+// Returns a DOM Fragment equivalent to "<em>Hello World</em>".
+s.sanitize("<em>Hello World</em><script>alert('surprise');</script>");
+```
+
+## Can I use the Sanitizer in my app?
+
+Not yet. Current browser support is preliminary and incomplete and still
+diverges in several details. And the specification itself is still in flux.
+We expect to be working through these issues in the coming months.
+
+If you wish to try it out already:
+
+* Firefox: Enable the `sanitizer.enabled` preference.
+* Chromium / Chrome: Start the browser with the
+  `--enable-blink-features=SanitizerAPI` flag.
+
+## Do I have to supply my own Sanitizer configuration? What is the default?
+
+The default configuration will block all content that causes script execution.
+unknown elements, plus a handful of deprecated elements.
+It is safe against any direct XSS.
+
+```js
+const default_sanitizer = new Sanitizer();
+
+// Returns a DOM Fragment equivalent to "<em>Hello World</em>".
+default_sanitizer.sanitize("<em>Hello World</em>");
+
+// Complex formatting is allowed by default. This will pass unchanged.
+default_sanitizer.sanitize("<div id='hello'><span style='font-weight: bold'><Hello World</span></div>");
```

Comment:
Is the `<` before "Hello" a typo, or an example? If it's an example, I'm not sure "pass unchanged" is a good description?

Reply:
:+1:  to what Mike said. We shouldn't promise "unchanged", but either functionally-equivalent or XSS-free.

Reply:
Done.

`<` was typo. Also changed the wording.